magento/magento2
# Fixed issue Mywishlist sidebar 20983

### Description (*)

Magento Luma theme sidebar my wish list design improvement


### Fixed Issues (if relevant)

1. magento/magento2#20984: Magento Luma theme sidebar my wish list design improvement



### Manual testing scenarios (*)

1. Add to wishlist any product and check the left sidebar in tablet mode (768px, portrait).


### Contribution checklist (*)
